docs/.gitignore

```diff
@@ -0,0 +1 @@
+_*
```

Comment:
It's uncommon to see a `.gitignore` outside of the root of the repo. Consider adding something similar to the following to the project's `.gitignore`.
```
docs/_build
docs/build
```